runtime/js/40_testing.js

```diff
@@ -297,7 +297,7 @@ finishing test case.`;
   }
 
   function formatError(error) {
-    if (error.errors) {
+    if (error.errors && typeof error.errors.map === "function") {
```

Comment:
Although extremely unlikely, it is also possible that it might have an `error.errors.map` that's a function that is not an array map function.

I think we should explicitly check if  `error instanceof AggregateError` here instead and import that from `window.__bootstrap.primordials`. That would also make it more clear what this code was doing. Alternatively we could check that `error.errors instanceof Array`.

Reply:
I went for `error instanceof AggregateError`.